spec/system/authentication_system_spec.rb

```diff
@@ -26,8 +26,8 @@
     context 'with invalid credentials' do
       it 'does not allow to sign in' do
         click_link(I18n.t('common.button.log_in')) # rubocop:disable Capybara/ClickLinkOrButtonStyle
-        fill_in('Email', with: user.email)
-        fill_in('Password', with: password.reverse)
+        fill_in(:user_email, with: user.email)
+        fill_in(:user_password, with: password.reverse)
```

Comment:
I don't know the details here tbh, but usually I prefer using real labels. This does not seem to be possible anymore.

Reply:
We could still use a real label, for sure. However, I simply went for consistency: In other tests, we used the symbol already (which is not a `I18n.t()` label, but the ID of the HTML element.

We could change it, for sure, but then we should do so globally. For now, I would rather have it uniformed with the other occurrences.